Publish /en/lessons/teach-history-and-languages-with-strategy-game

anisa-hawes opened the pull request:
Preparing files for publication.

Closes #3559 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - ~~[ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.~~

anisa-hawes committed:
Update ph_authors.yml

Add bio for Corey Stephan.

anisa-hawes committed:
Upload /images/teach-history-and-languages-with-strategy-game

Upload image directory.

anisa-hawes committed:
Upload teach-history-and-languages-with-strategy-game-original.jpg

Upload original avatar.

anisa-hawes committed:
Upload teach-history-and-languages-with-strategy-game.png

Upload gallery avatar.

anisa-hawes committed:
Create teach-history-and-languages-with-strategy-game.md

Create Markdown file for new EN original lesson.

anisa-hawes commented:
Hello Alex @hawc2,

I’ve staged teach-history-and-languages-with-strategy-game for publication. You can [read it through in Preview](https://deploy-preview-3560--ph-preview.netlify.app/en/lessons/teach-history-and-languages-with-strategy-game) and let me know if you note anything which you'd like to adjust.

Thank you.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Adjust `date:`

hawc2 commented:
@anisa-hawes this looks great overall. The avatar is perfect, good choice. 

There are some minor changes I'd suggest for the abstract. 

- [x] It should be "Teaching" instead of "Teach" - going by our style for other lesson titles, and my preference. 

- [x] Instead of "It presents" (since I'm not a fan of "it" in an abstract), it could be, with a couple tweaks in the sentence: "You will learn how to develop a multisensory, interactive environment for teaching and learning ancient history and pre-modern languages." Note I think "project-based" is weak and what's important is the "interactivity" of the media being created/used.

- [x] The 0 in 0 A.D. looks more like the letter O. Is that a typo, and is that typo elsewhere in the lesson? Or is there any way to make the o look as tall as the capital A.D.? If not, I guess this is what we look forward to with a website redesign. I see it happens in some of the other Headings, so it must be that...

- [x] This is something I should've caught before, but the very first words of the lesson are: "[0 A.D.: Empires Ascendant](https://play0ad.com/)". Everywhere I've looked it's simply called 0 A.D. The link doesn't lead to anything that refers to it as Empires Ascendant. Maybe that term can be brought up later, but as the very first words, it is a little misleading I fear. 

- [x] Note that Empires Ascendant is only used one other time in the lesson, and no mention of the name is given in that instance either. In my opinion, it's easier to remove it. If we keep it, the lesson would be better off with a sentence considering the meaning of that title. 

The meaning of 0 A.D. is something that could've been worth explaining as well, but it's not necessary: it will be pretty apparent to most people what zero A.D. means, and they can always look that up. If anything, considering the way 0 appears in the title, and the general way I've failed to pronounce the title of this game correctly to myself as I've read the lesson over the last year (I kept thinking it was "O.A.D", not "0 A.D."), I'm mostly concerned that what's confusing is that symbol (O or 0), and it could even just help if we wrote out the phonetics, "Zero A.D.", in parentheses in the body of the first paragraph to make it completely explicit: "0 A.D. (phonetically, "Zero A.D."). Complicating matters, it's worth noting Wikipedia also has a footnote about the title: "Stylized with the 0 replaced with a [sun cross](https://en.wikipedia.org/wiki/Sun_cross)."

Otherwise, this lesson looks ready to go to me. I will say, though, that it's very detailed, complex, and pretty different from our usual lesson, so it may be worth asking the author to look it over one last time to see if they catch anything subtle I'm missing. 

This one will be really popular with communities that Programming Historian isn't well known in, so that's exciting!

charlottejmc commented:
Hi @hawc2,

Just to jump in about the 0 A. D. title. It's definitely a 0 and not an o in the Markdown file, and I've checked on Google Fonts: unfortunately, that is how [Crete Round](https://fonts.google.com/specimen/Crete+Round) displays zeros... I agree that it looks really odd and misleading here.

We could try forcing a 0 with a backslash using `0&#x0338;`, but we'd have to test how this looks on the preview, as GitHub displays it like this 0&#x0338;.

We could use the `&#x2A01;` HTML code which displays like this &#x2A01; (maybe just once at the start, explaining that it is pronounce as 'Zero'?) or embed a small icon image of the symbol, as we've successfully done in the Map Warper lessons, like this:

`The lock between the two windows helps guide the assignment and movement of the control points. ![Zoom lock icon](/images/introduccion-map-warper/es-or-introduccion-map-warper-icon6.png)The **Zoom lock** option allows you to zoom in on the maps in both windows simultaneously. This is useful to place and verify the control points. Clicking on the lock again will unlock it. ![Pan icon](/images/introduccion-map-warper/es-or-introduccion-map-warper-icon7.png)The **Pan** function allows you to pan to different sections of the images, separately.`

See how it looks [here](https://programminghistorian.org/en/lessons/introduction-map-warper#georeference-your-map).

According to the site, the first part of the game "Empires Ascendant" covers the period from 500 BC to 1 BC and the second part (which hasn't been released yet) will be based from 1 AD to 500 AD.

The game developers seem to use '0 A.D.' and '0 A.D. Empires Ascendant' rather interchangeably, but I agree that it makes more sense to use only 0 A.D. in our case, especially if they release the second half, which would be under a different name (some sources suggest 'Empires Besieged').

And just as a small note, I think Corey does explain what 0 A.D. means in this section:

"The name 0 A.D., however, seems to serve as a reminder that, while the game has many true-to-life elements, it remains a game. The [Han Dynasty](https://perma.cc/3VCW-DRGS) and the [Spartan Empire](https://perma.cc/36VZ-MCJ5), for example, were never at war with each other, nor did they even overlap temporally. The historical accuracy of 0 A.D. is not in the reality of specific civilizations encountering each other but, rather, in the details of each civilization itself. The Spartans are designed with the militaristic, economic, domestic, cultural, and religious trappings from their peak as the Spartan Empire, as are the Hans in their own way. The game might be best understood as an imagined clashing of multiple real, prominent civilizations between 500 BCE and 500 CE. 0 A.D. is [the year that never was](https://perma.cc/MRH2-MUC7): a fictional nexus between civilizations and events that really were."

Actually, reading this over now, I notice two things:
- Corey mentions civilizations from 500 BCE to 500 CE, which includes the 'Empires Besieged' time period. Although it is the game's intention to cover the entire period, the years 1 - 500 CE haven't been released yet.
- It may be confusing to suddenly use BCE and CE here, at least without explaining them as 'Before Common Era' and 'Common Era'.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Test html code.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

- Adjust title
- Adjust abstract
- Remove 'Empires Ascendant'
- Replace 0 with 0&#x0338; throughout (html code)

anisa-hawes commented:
Thank you, Alex @hawc2 and Charlotte @charlottejmc!

I have now checked off each of the tasks/suggestions Alex has made above:

- [x] Adjusted the title
- [x] Adjusted the abstract
- [x] Removed 'Empires Ascendant' at line 1
- [x] Replaced 0 with `0&#x0338;` throughout (html code)

For the abstract, I have suggested:

This lesson _demonstrates_ how to prepare a geographically accurate historical battle scene in the free and open source computer game 0&#x0338; A.D. You will learn to _use_ a multisensory, _interactive_ environment for teaching or learning ancient history and pre-modern languages.

The word _demonstrate_ allows us to avoid using the metaphor of sight (_show_ how to do something). Charlotte and I agree that the verb _to use_ might be preferable than _to develop_ in this case. The game exists, and the learning actions are adapting and using it for teaching. 

I will ask Corey about taking an opportunity to review it for a final time in Preview. I will raise Charlotte's question about whether it could be useful to define BCE and CE as 'Before Common Era' and 'Common Era'.

hawc2 commented:
@anisa-hawes That all sounds good to me, agreed on "demonstrates" and "uses" 

I did notice as @charlottejmc points out that the lesson does explore the game's name and it's meaning, but it is very indirect and a bit buried. I think if you look at the Wikipedia entry's first couple paragraphs, it shows what the most essential details are to share with anyone about the game upfront. 

I am still a bit confused about the dating. When I read about the game online, it said it's called 0 A.D. because the game is focused on what happens between 1 and 500 A.D. It seems to me a few different expansions of the game including the B.C. eras and the Empires Ascendant label are being conflated onto the base game? I'm sure it's a complicated development history and games can get really confusing with expansions or mods, and that's hard to capture, but it would be helpful to make these key details of the base game more plain and upfront to better situate for the reader what historical period this game is supposed to be representing. 

It might just be a matter of taking a couple sentences from wikipedia to use to provide basic setting. But it's worth making sure Corey agrees with any of those edits, since I might be misunderstanding something. 

Alot of this is a relic of the feedback I gave on the final stages of drafting, where there wasn't really a basic introduction to the game before the reader was immersed in the mechanics. The lesson does that better now, but it still feels a bit like it introduces the essential parts of the game in a roundabout way.

hawc2 commented:
@charlottejmc just noticing now you said: "Although it is the game's intention to cover the entire period, the years 1 - 500 CE haven't been released yet." I misstated in my post just now - here's what it says on Wikipedia, that I think we could include almost verbatim: "It is a historical war and economy game focusing on the years between 500 BC and 1 BC, with the years between 1 AD and 500 AD planned to be developed in the future.[[](https://en.wikipedia.org/wiki/0_A.D._(video_game)#cite_note-SERIOUSHISTORY-3)"

hawc2 commented:
@anisa-hawes @charlottejmc I guess the main point is that the game was named 0 A.D. because it tells the story of what led up to 0 A.D. in the preceding 500 years. I kept thinking 0 A.D. was the start point of the game, not the end point. That seems important to emphasize about what historical knowledge it is teaching.

Definitely confusing all around! Thanks for figuring out the 0 - O solution. Looks good to me, as does the phonetic explanation at the start of the lesson.

charlottejmc commented:
Hi @hawc2,

From what I understand, the goal is to eventually cover 500 BC - 500 AD. In this context, the name 0 A.D. highlights the fictional aspect of the game, because it's a year which never actually existed within that time frame (we go directly from 1 BC to 1 AD).

However, because they've only developed the first portion 500 BC - 1 BC (and this is the part of the game called Empires Ascendant), it doesn't really make sense yet. 

I've added a short clarification in the first paragraph, and I also decided to change BCE/CE to B.C. and A.D. (keeping them punctuated to be consistent with the game title).

charlottejmc committed:
Update teach-history-and-languages-with-strategy-game.md

Minor edits

anisa-hawes commented:
Thank you Alex @hawc2, and thank you Charlotte @charlottejmc.

[I've written](https://github.com/programminghistorian/ph-submissions/issues/613#issuecomment-2898677458) to Corey and Caio, inviting them to review these final adjustments, and read the lesson one more time before publication.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Update `date:`

hawc2 commented:
Thanks so much @charlottejmc for that clear explanation. Makes sense, and it all seems worth clarifying, including that idea of the 'fictional' 0 A.D. I've learned more basic history than I should admit through clarifying the title of this game - my mind is slightly blown at the fact that 0 A.D. does not get counted from 1 B.C. to 1 A.D.!

historical-theology commented:
Dear @hawc2 , @charlottejmc , and @anisa-hawes ,

Thank you for your work in these final preparations, and thank you for providing me with the opportunity to do one last painstakingly close reading of the lesson in its latest form, which includes many recent edits from each of us.

First, I think that what you folks have asked about clarifying what "B.C.E." and "C.E." means points to a problem with the opening paragraph, namely, that we use those 2 terms at all. Please change them to "B.C." and "A.D.," respectively, since shifting between 2 different standards for naming dates throughout the lesson (with the lesson title itself necessarily including "0 A.D.") strikes me as needlessly confusing. We already use "B.C." and "A.D." everywhere else in the lesson.

Second, the parenthetical that @charlottejmc added to the first paragraph ("between 1 A.D. and 500 A.D." and so on), and the broader discussion in this editorial thread about the game coming in 2 parts, is not actually relevant to our lesson. The game developers' priorities have shifted with time, and the game already includes civilizations that span the 1 B.C. to 1 A.D. division (e.g. the Hans). I respectfully ask that you remove that parenthetical.

Third, here is a sequential checklist of quick, minor edits that seem to me to be worthy of making before publication. I apologize about this list's length. Many of these small problems have come during the long editorial process through which the lesson has gone, especially the latest rounds.

Note that I have requested (in the course of this sequential checklist) the removal of all contractions that you, dear editors, added throughout the lesson in your late editing. I did not write the lesson with any contractions, so their sporadic insertion comes across to me as interrupting the lesson's flow. I do not know, however, if you might have some important reason for keeping them, such as part of a special standard at PH.

- [ ] "However, you may choose" CHANGE TO "**Yet,** you may choose" [traditionally unaccepted introductory "However"]
- [ ] "control over the entire geographic region" CHANGE TO "control over the entire geographic region **in which you are playing**" [clarification]
- [x] "Begin drafting your own stand-alone" CHANGE TO "Begin **outlining** your own stand-alone" [accuracy]
- [x] "which you'll import" CHANGE TO "which **you will** import" [remove contraction]
- [x] "for example by incorporating new civilizations" CHANGE TO "**such as** by incorporating new civilizations" [comma problem]
- [x] "to the large number of mods available" CHANGE TO "to the large number of mods **that are now available**" [clarification]
- [x] "a member of the History Department" CHANGE TO "**then** a member of the **so-called** History Department" [clarifies that this was something of a joke -- and that it dates back to 2006]
- [x] "you'll find which new translations" CHANGE TO "**you will** find which new translations" [remove contraction]
- [ ] "might work the Latin translation, and then play" CHANGE TO "might work **on** the Latin translation, and then **they could** play" [missing preposition + comma splice]
- [x] "You'll find dozens to hundreds" CHANGE TO **"you will** find dozens to hundreds" [remove contraction]
- [x] "click-to-install community mods which" CHANGE TO "click to install community mods **that**" [which/that mishap]
- [x] "You'll need to enable the mod(s)" CHANGE TO **"you will** need to enable the mod(s)" [remove contraction]
- [x] "(custom Pyrogensis package" CHANGE TO "custom **Pyrogenesis** package" [spelling]
- [x] Since we removed the earlier discussion in the lesson of Pyrogenesis being the custom game engine for 0 A.D., which is also open source, after "You can even do this in the command line with pyrogenesis..." a quick parenthetical should be added, like so: "(Pyrogenesis is 0 A.D.'s built-from-scratch, open source engine.)
- [x] "On Unix-like OS," CHANGE TO "On **a** Unix-like OS" [indefinite article]
- [x] "ninth and tenth centuries CE" CHANGE TO "ninth and tenth centuries **A.D.**" [discussion above]
- [x] ", which took place in a location" CHANGE TO "The battle should have taken place in a location" [new sentence prevents subject-predicate confusion]
- [x] "late Republican Romans, Gauls and Britons" CHANGE TO "late Republican Romans, Gauls**,** and Britons" [the Oxford comma is used everywhere else]
- [x] "topography databases to choose from," CHANGE TO "topography databases **from which you could choose**," [removes clause's dangling preposition]
- [x] "let's choose a 512x512" CHANGE TO "**let us** choose a 512x512" [remove contraction]
- [x] "You'll probably need to experiment with the resulting" CHANGE TO "**You will** probably need to experiment with the resulting" [remove contraction]
- [x] "Would you like to quit 0 A.D. and open the Scenario Editor?" CHANGE TO "**Are you sure you want** to quit 0 A.D. and open the Scenario Editor?" [fixes mismatch between quotation and what is shown in the screenshot, which was added during final revision]
- [x] "If you've followed the Mount Boevray example" CHANGE TO "If **you have** followed the **Mont** Beuvray example" [remove contraction + spelling]
- [x] "Let's add deer and other wildlife" CHANGE TO "**Let us** add deer and other wildlife" [remove contraction]
- [x] "Although you'll certainly adapt your use of these tools" CHANGE TO "Although **you will** certainly adapt your use of these tools" [remove contraction]
- [x] "Don't change any information in the Random Map fields" CHANGE TO "**Do not** change any information in the Random Map fields" [remove contraction]
- [x] "filter before selecting the map you've just created" CHANGE TO "filter before selecting the map **you have** just created" [remove contraction]
- [x] "limited by the Pyrogenesis game engine's own limits" CHANGE TO "limited by **the game engine's** own limits" [since the fuller discussion of Pyrogenesis was removed in a revision, the term does not seem to belong here any more -- see also the edits about this above]
- [x] "Further, 0 A.D.: Empires Ascendant is fully" CHANGE TO "Further, 0 A.D. is fully" [without "Empires Ascendant," in keeping with the discussion earlier in this editorial thread]

I wish for all of you to know that I am immensely pleased by how this tutorial has come together. The open review process of the Programming Historian is the first that I have had, and I have found it to have some real advantages over other methods of working through peer review.

With that, I rest my case, and I look forward to seeing this tutorial in its published form. As the author, I am, I write again, (highly) satisfied.

Thank you, and I hope to work with (all of) you again in the future,
Corey

charlottejmc committed:
Update teach-history-and-languages-with-strategy-game.md

Author's suggested edits

charlottejmc committed:
Update teach-history-and-languages-with-strategy-game.md

add commas

charlottejmc commented:
Dear @historical-theology,

Thank you very much for your close read and comments. Thank you for pointing out the contractions, which I admit are all mine! You're completely right that they stood out from your original text – this is something I'll be more careful about in the future. 

I've integrated the edits you suggest with just three minor changes:
- [x]  Yet, you may choose to follow these steps -> You may, however, choose to follow these steps
- [x] control over the entire geographic region **in which you are playing** -> I chose not to add the clarification, as it already seems very clear to me in this context. The sentence is already quite long as it is, so I fear it would end up being unnecessarily wordy.
- [x] might work on the Latin translation, and then they could play -> I made a simpler change by removing the comma altogether: "might work on the Latin translation and then play"

I hope this is OK with you.

Thank you for your kind words, Corey. We always appreciate feedback on a workflow that, while carefully crafted over many years, is still a work in progress. It's great to hear that you've had a positive experience with it. It was a pleasure collaborating with you too, and I know we are all excited to publish your lesson very soon!

historical-theology commented:
@charlottejmc 

Thank you for implementing the edits that I requested.

I like what you did with those 3 alternative changes.

The only remaining thing that I can see (the 1 item that I forgot to include in the checklist above) is a lingering extra space at the end of endnote 3 (between 3 and 4).

Otherwise, from my view as the author, _imprimatur._

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Adjust line breaks in endnotes.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Adjust line breaks in endnotes.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

test fullstop, line 392.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

-Remove fullstop (test)
- Adjust line break in endnotes

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Adjust line breaks.

anisa-hawes commented:
Many thanks, once again Corey @historical-theology. The apparent errant space following endnote 3 is very strange because I didn’t find an extra space there... I have tried to adjust this in several ways, but unfortunately have not been successful.

I think it is a small oddity of the Markdown which we may need to accept this time. I suspect it is related to the bulleted list contained within this endnote. I notice that in endnote  2, where the bulleted list is followed by a sentence, the space/line break isn't introduced before the 'return' arrow.

anisa-hawes committed:
Update teach-history-and-languages-with-strategy-game.md

Update `date:`

hawc2 reviewed (approved):
@anisa-hawes looks ready to go to me. @historical-theology thanks for this detailed last review, and for your steadfast work on this lesson developing it over the last couple years! It came out excellent

anisa-hawes commented:
Many thanks to all! We will publish this next Wednesday ✨
